packages/completer-extension/schema/manager.json

```diff
@@ -15,8 +15,8 @@
       },
       "additionalProperties": false,
       "default": {
-        "CompletionProvider:context": 100,
-        "CompletionProvider:kernel": 200
+        "CompletionProvider:context": 550,
+        "CompletionProvider:kernel": 500
```

Comment:
Does higher rank = preferred? If so, kernel should have a higher rank, right? This is how it used to be. It may be also why the integration tests are failing.

Reply:
Yes, rank = preferred. I pushed a commit to fix it.